docs/heasarc/heasarc.rst

```diff
@@ -264,15 +264,31 @@ tables listing most recent INTEGRAL data.
     ...                              radius='361 degree', time="2021-02-01 .. 2030-12-01",
```

Comment:
The Integral science window database table is still being updated so the timeframe in this query should probably be modified so that new updates to the database table don't constantly change this output.
https://heasarc.gsfc.nasa.gov/W3Browse/integral/intscw.html shows it was last updated today.